torchsummary/tests/unit_tests/torchsummary_test.py

```diff
@@ -39,9 +43,22 @@ def test_multiple_input_types(self):
         input1 = (1, 300)
         input2 = (1, 300)
         dtypes = [torch.FloatTensor, torch.LongTensor]
-        total_params, trainable_params = summary(model, [input1, input2], device="cpu", dtypes=dtypes)
+        total_params, trainable_params = summary(
+            model, [input1, input2], device="cpu", dtypes=dtypes)
         self.assertEqual(total_params, 31120)
         self.assertEqual(trainable_params, 31120)
 
+
+class torchsummarystringTests(unittest.TestCase):
+    def test_single_input(self):
+        model = SingleInputNet()
+        input = (1, 28, 28)
+        result, (total_params, trainable_params) = summary_string(
+            model, input, device="cpu")
+        self.assertEqual(type(result), str)
+        self.assertEqual(total_params, 21840)
+        self.assertEqual(trainable_params, 21840)
+
```

Comment:
So I reduced this test class into single test case. The reason I remained (although it is somewhat duplicated) test oracles for parameters is to denote the interface of `summary_string` for all return variables. Thanks for the additional suggestion! @Naireen